src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -0,0 +1,142 @@
+import moment from 'moment'
+import React, { useState } from 'react'
+import { Button, Table } from 'react-bootstrap'
+import { SMALL_USER_CARD } from '../utils/SynapseConstants'
+import {
+  AccessApprovalSearchRequest,
+  AccessApprovalSearchSort,
+  AccessApprovalSortField,
+  Direction,
+  SortDirection,
+} from '../utils/synapseTypes'
+import Typography from '../utils/typography/Typography'
+import UserCard from './UserCard'
+import { PRODUCTION_ENDPOINT_CONFIG } from '../utils/functions/getEndpoint'
+import { useSearchAccessApprovalsInfinite } from '../utils/hooks/SynapseAPI/dataaccess/useAccessApprovals'
+import { SynapseSpinner } from './LoadingScreen'
+import SortIcon from '../assets/icons/Sort'
+import { formatDate } from '../utils/functions/DateFormatter'
+
+export type AccessApprovalsTableProps = {
+  accessorId: string
+  accessRequirementId?: string
+}
+export const AccessApprovalsTable: React.FunctionComponent<
+  AccessApprovalsTableProps
+> = ({ accessorId, accessRequirementId }) => {
+  const [isDescending, setIsDescending] = useState(false)
+  const [sort, setSort] = useState<AccessApprovalSearchSort[]>()
+
+  const searchRequest: AccessApprovalSearchRequest = {
+    accessorId,
+    accessRequirementId,
+    sort,
+  }
```

Comment:
Couple of (fairly picky) notes but hopefully you agree it helps clean it up:
* We're not using more than one sort, so we don't need to store an array
* isDescending can be determined from the sort object, so let's get rid of it
* No reason not to have a default sort state, which will allow us to not worry about an undefined value
* We should memoize the final request object, just to ensure we don't dispatch unnecessary requests

```suggestion
  const [sort, setSort] = useState<AccessApprovalSearchSort>({ field: AccessApprovalSortField.MODIFIED_ON, direction: SortDirection.DESC  })

  const searchRequest: AccessApprovalSearchRequest = useMemo(() => ({
    accessorId,
    accessRequirementId,
    sort: [sort],
  }), [accessorId, accessRequirementId, sort])
```